ee/libcglue/src/glue.c

```diff
@@ -637,7 +637,7 @@ int _gettimeofday(struct timeval *tv, struct timezone *tz)
 
 #ifdef F__times
 clock_t _times(struct tms *buffer) {
-	clock_t clk = GetTimerSystemTime() / (kBUSCLK / CLOCKS_PER_SEC);
```

Comment:
Even being the same value, I think that using `CLOCKS_PER_SEC` could be "confusing"